packages/react/src/app-layout/AppLayoutHeaderBrand.tsx

```diff
@@ -9,24 +9,138 @@ import {
 } from '../core';
 import { Text } from '../text';
 import { Box } from '../box';
+import { Stack } from '../stack';
+import { AppLayoutHeaderProps } from './AppLayoutHeader';
+
+const GAP_REM = 1.5;
 
 export type AppLayoutHeaderBrandProps = {
 	href: string;
 	logo: JSX.Element;
+	hasAccountDetails: boolean;
 	heading: string;
-	subLine?: string;
 	badgeLabel?: string;
+	dividerPosition?: AppLayoutHeaderProps['dividerPosition'];
+	secondHref?: AppLayoutHeaderProps['secondHref'];
+	secondLogo?: AppLayoutHeaderProps['secondLogo'];
+	subLine?: string;
 };
 
 export function AppLayoutHeaderBrand({
-	heading,
-	subLine,
 	badgeLabel,
+	dividerPosition,
 	logo,
+	heading,
+	hasAccountDetails,
 	href,
+	secondHref,
+	secondLogo,
+	subLine,
 }: AppLayoutHeaderBrandProps) {
 	const Link = useLinkComponent();
-	return (
+
+	return logo && secondLogo ? (
+		<Flex
+			css={{ overflow: 'hidden' }}
+			flexDirection={{ xs: 'column', lg: 'row' }}
+			flexWrap={{ xs: 'wrap', xl: 'nowrap' }}
+			gap={GAP_REM}
+			inline
+			paddingY={1}
+		>
+			<Flex
+				css={{
+					marginRight:
+						hasAccountDetails && dividerPosition === 'after'
+							? mapSpacing(GAP_REM)
+							: 0,
+				}} // Create a gap when there is right content
+				flexDirection={{ xs: 'column', sm: 'row' }}
+				flexShrink={0}
+				gap={GAP_REM}
+			>
+				<Flex
+					as={Link}
+					color="text"
+					css={{
+						' img, svg': { height: '3.75rem' },
```

Comment:
Why are we hard-coding this height?

Reply:
Good question, I'm not sure. This is just copy/paste from existing in this file.

Reply:
So it was. It's really hard to tell from the diffs